packages/driver/cypress/e2e/dom/visibility.cy.ts

```diff
@@ -53,7 +53,7 @@ describe('src/cypress/dom/visibility', () => {
       expect(fn()).to.be.true
     })
 
-    it('returns false if window and body < window height', () => {
```

Comment:
why are the visibility tests removed?

Reply:
This is work to undo: https://github.com/cypress-io/cypress/pull/27860 in order to fix a regression caused by that (https://github.com/cypress-io/cypress/issues/28150)